docs/config.md

```diff
@@ -133,30 +133,34 @@ If specified, a field's value must be set to be a hex representation of a RGB co
 
 ### Component Styles
 
-To define application's component styles, user needs to specify **all of the below fields**:
+To define application's component styles, the user can specify any of the below fields:
 
 - `block_title`
+- `border`
 - `playback_track`
 - `playback_album`
 - `playback_metadata`
 - `playback_progress_bar`
 - `current_playing`
 - `page_desc`
 - `table_header`
+- `selection`
 
 A field in the component styles is a `Style` struct which has three optional fields: `fg`, `bg` and `modifiers`. `fg` and `bg` can be either a palette's color (string in pascal case) or a custom RGB color using the following format: `fg = { Rgb { r = ..., g = ..., b = ... } }`. `modifiers` can only be either `Italic` or `Bold`.
 
 Default value for application's component styles:
 
 ```toml
 block_title = { fg = "Magenta"  }
+border = {} # uses the palette's fg and bg color
 playback_track = { fg = "Cyan", modifiers = ["Bold"] }
 playback_album = { fg = "Yellow" }
 playback_metadata = { fg = "BrightBlack" }
 playback_progress_bar = { bg = "BrightBlack", fg = "Green" }
 current_playing = { fg = "Green", modifiers = ["Bold"] }
 page_desc = { fg = "Cyan", modifiers = ["Bold"] }
 table_header = { fg = "Blue" }
+selection = {} # if not set, this is bold and reverses fg and bg color
```

Comment:
```suggestion
selection = { modifiers = ["Bold", "Reverse"] } 
```

Note that this requires you to add a new enum variant `StyleModifier::Reverse`

Reply:
implemented in https://github.com/aome510/spotify-player/pull/118/commits/591568db595cbe27b517b96e07f6837008b97272